Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR_test.py

```diff
@@ -0,0 +1,149 @@
+from CommonServerPython import *
+from typing import Dict, Any, List
+
+MOCK_URL = 'https://nonexistent-domain.com/'
+
+
+def test_connection_success(requests_mock: Any) -> None:
+    '''Command for test-connection'''
+
+    expected_mocked_command_result: str = 'ok'
+    requests_mock.get(f'{MOCK_URL}/api/app/info')
+    assert expected_mocked_command_result
```

Comment:
The tests overall seem good, my only suggestion is to use the `client.test_conectivity` in the test_connection_* tests. 
(You just asserted that the expected_mocked_command_result is not None which is always true)